keyboards/zeix/acidandco/stellaron/keyboard.json

```diff
@@ -0,0 +1,482 @@
+{
+    "manufacturer": "Acid & Co",
+    "keyboard_name": "Stellaron TKL",
+    "maintainer": "itsme-zeix",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "GP27",
+        "on_state": 1,
```

Comment:
```suggestion
```

Reply:
The hardware configuration has the reverse behaviour for this (tested and confirmed on PCB).

Reply:
`1` is the default, it can be safely removed.

Reply:
Gotcha! Have made the change.